pkg/dcmtime/common.go

```diff
@@ -13,14 +13,14 @@ import (
 // Example: to test whether seconds should be included, you would:
 // isIncluded(PrecisionSeconds, [caller-passed-limit])
 func isIncluded(check PrecisionLevel, precision PrecisionLevel) bool {
```

Comment:
nit should precision be "limit" here based on terminology in L11?

```suggestion
func isIncluded(check, limit PrecisionLevel) bool {
```

Reply:
makes sense!